### PR TITLE
Added documentation for CSS Text Effects

### LINKED ISSUE #7648: CSS Text Effects
### Description

I will provide descriptive explanation regarding css text effects in documentation format.

### Domain

Frontend Dev HTML/CSS/JS

### Type of Contribution

Documentation

### Code of Conduct

- [X] I follow [Contributing Guidelines](https://github.com/girlscript/winter-of-contributing/blob/main/.github/CONTRIBUTING.md) & [Code of conduct](https://github.com/girlscript/winter-of-contributing/blob/main/.github/CODE_OF_CONDUCT.md) of this project.

### PR DESCRIPTION
**Pull request by Jinal4502**
<hr>

## Description 📜

I have provided descriptive explanation regarding css text effects in documentation format.
Fixes #7648 

<hr>

## Type of change 📝

- [ ] Au#dio (Should be in mp3 format Includes speech clarity, Concise ,Low distortion)
- [ ] Vi#deo (Animations, screen-recordings, presentations and regular explanatory films are all possibilties etc)
- [x] Doc#umentation (Content Creation in the form of codes or tutorials)
- [ ] Other (If you choose other, Please mention changes below) 

<hr>

## Domain of Contribution 📊

- [ ] Android Dev #(Flutter)
- [ ] Android Dev #(Java)
- [ ] Android Dev #(Kotlin)
- [ ] Backend Dev #(Java)
- [ ] Backend Dev #(.NET)
- [ ] Backend Dev #(PHP)
- [ ] Backend Dev #(Python)
- [ ] C/#CPP
- [ ] Competitive #Programming
- [ ] Cyber #Security
- [ ] DS#A
- [ ] Data#base
- [ ] Datascience with #Python
- [ ] Datascience with #R
- [x] Frontend Dev #HTML/CSS/JS
- [ ] Frontend Dev #React/Angular/Vue
- [ ] Go#lang
- [ ] Interview #Prep
- [ ] Java_#Domain
- [ ] JavaScript_#Domain
- [ ] ME#RN
- [ ] Machine #Learning
- [ ] Open #Source
- [ ] Python_#Domain
- [ ] Ru#st
- [ ] Statis#tics
- [ ] UI/#UX

<hr>

## Checklist ✅

- [x] I follow [Contributing Guidelines](https://github.com/girlscript/winter-of-contributing/blob/main/.github/CONTRIBUTING.md) & [Code of conduct](https://github.com/girlscript/winter-of-contributing/blob/main/.github/CODE_OF_CONDUCT.md) of this project.
- [x] I have performed a self-review of my own code or work.
- [x] I have commented my code, particularly in hard-to-understand areas.
- [x] My changes generates no new warnings.
- [x] I'm GWOC'21 contributor

<hr>

## Screenshots / Gif (Optional) 📸

<hr>

### CODE REVIEW

**Review by kashishahuja2002 — CHANGES_REQUESTED** [review]
image
Plagiarism detected, refactor it.

**Review by kashishahuja2002 — CHANGES_REQUESTED** [review]
Add some code and SS for better understanding.

**Comment by Jinal4502** [comment]
Sure, will add it

**Comment by Jinal4502** [comment]
@kashishahuja2002 I have added code and output screenshots, please check it, thank you!

**Review by kashishahuja2002 — APPROVED** [review]
LGTM. Tag the supervisor of your batch for further reviewing and merging the PR.

File: Web_Development/FrontEnd/CSS_Text_Effects/text_effects.md [hunk 1]
```diff
@@ -0,0 +1,271 @@
...
+
+<b>clip: </b>Clips the overflowed text.
+
+<b>ellipsis: </b>Displays an ellipsis (…) or three dots to show the clipped text.
+
+<b>initial: </b>Iniializes CSS property to its default value.
+
+<b>string: </b>Displays clipped text to the user using the string of programmer's choice. But this feature is only accessible in Firefox browser.
+
+<b>inherit: </b>It will set the text-overflow value to its parent element's value.
```
Comment (2024-SANDHYA) [inline Web_Development/FrontEnd/CSS_Text_Effects/text_effects.md:31]: Not the way to write syntax , either use ``` or use numbering.

File: Web_Development/FrontEnd/CSS_Text_Effects/text_effects.md [hunk 2]
```diff
@@ -0,0 +1,271 @@
...
+    <h2 id="tb"> writing-mode: horizontal-tb; </h2>  
+    <h2 id="lr" style= "height: 300px;"> writing-mode: vertical-lr; </h2><br>  
+    <h2 id="rl" style= "height: 300px;"> writing-mode: vertical-rl; </h2>  
+  </body>
+  </center>    
+</html>
+```
+### Output:
+
+<img src='https://user-images.githubusercontent.com/65852362/141079060-63ed08b2-7981-4c90-a83e-dfc572df1cc0.png'>
```
Comment (2024-SANDHYA) [inline Web_Development/FrontEnd/CSS_Text_Effects/text_effects.md:271]: No image credits are given, add them.

**Review by 2024-SANDHYA — CHANGES_REQUESTED** [review]
No reference links & image credits given.

**Comment by Jinal4502** [comment]
@2024-SANDHYA, I have made the changes you mentioned, please check.

File: Web_Development/FrontEnd/CSS_Text_Effects/text_effects.md [hunk 3]
```diff
@@ -0,0 +1,285 @@
...
+<b>clip: </b>Clips the overflowed text.
+
+<b>ellipsis: </b>Displays an ellipsis (…) or three dots to show the clipped text.
+
+<b>initial: </b>Iniializes CSS property to its default value.
+
+<b>string: </b>Displays clipped text to the user using the string of programmer's choice. But this feature is only accessible in Firefox browser.
+
+<b>inherit: </b>It will set the text-overflow value to its parent element's value.
+
```
Comment (2024-SANDHYA) [inline Web_Development/FrontEnd/CSS_Text_Effects/text_effects.md:34]: If you can write this in a proper way, maybe numbering or use ``` ?

**Review by 2024-SANDHYA — CHANGES_REQUESTED** [review]
Please do the needful

File: Web_Development/FrontEnd/CSS_Text_Effects/text_effects.md (hunk 3 above)
Comment (Jinal4502) [reply Web_Development/FrontEnd/CSS_Text_Effects/text_effects.md:34]: sure, will do that.

**Comment by Jinal4502** [comment]
@2024-SANDHYA, I have corrected the presentation part of syntax's description, please check it, And I am really sorry, you had to mention so many changes, actually this was my first pr, that's why I didn't know some basics, thank you!!

**Review by 2024-SANDHYA — APPROVED** [review]
No issues @Jinal4502 You've done great work 🙌🏻 . We are here to guide you if you are stuck anywhere 🚀

**Pull request merged**